src/client/index.js

```diff
@@ -390,6 +391,17 @@ function mkcircuit(data, opts) {
     monitorview.on('change:pixelsPerTick', show_scale);
 }
 
+function exportCircuitAsSvg() {
+    // Create an artificial link and click it to download the temporary file
+    let downloadLink = document.createElement('a');
```

Comment:
The `file-saver` package is already used in this file, you could use that here too and avoid the artificial link weirdness here.